bool mStop
bool mGo
bool mTooClose
bool mTooFar
bool mTooClockwise
bool mTooCClockwise
